src/components/Drawer/Drawer.tsx

```diff
@@ -39,15 +37,19 @@ export const Drawer = ({
   onClose,
   title,
 }: DrawerProps): ReactElement => {
+  const closeIcon = useMemo(() =>
```

Comment:
can this be instantiated outside the component?

Reply:
I think so